### PR TITLE
RMET-192: Merge cordova apache version 4.0.0 

### PR DESCRIPTION
**Pull request by GFOutsystems**
## Description
Update with Apache fork 4.0.0 stable version.

## Type of changes

- [ ] Feature (change which adds functionality)
    - [x] BREAKING CHANGE (existing functionality will not work as expected due to new feature)
- [ ] Fix (change which fixes an issue)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to fix)
- [ ] Performance (change which improves performance)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to performance improvement)
- [ ] Refactor (non-breaking change that is neither feature, fix nor performance)
- [ ] Style (non-breaking change that only affects formatting and/or white-space)

## Components affected
- [x] Android platform
- [x] iOS platform
- [x] JavaScript
- [ ] OutSystems

## Tests
All iOS
- Valid URL (HTTPS, HTTP)
- Invalid URL
- PDF
- Cookie site
- Clean cash
- Button Title

## Checklist

- [x] Pull request title follows the format `RNMT-XXXX <title>`
- [ ] Tests have been created
- [ ] Code follows code style of this project
- [x] CHANGELOG.md file is correctly updated
- [x] Changes require an update to the documentation
    - [x] Documentation has been updated accordingly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-inappbrowser",
-  "version": "3.1.0-OS3",
+  "version": "4.0.0-OS4",
```
Comment (vmfo22) [inline package.json:3]: @GFOutsystems please reset the OS versioning. As we update the version of the plugin, we can just have OS telling us that there are custom changes applied.

File: plugin.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 <plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
            id="cordova-plugin-inappbrowser"
-      version="3.1.0-OS3">
+      version="4.0.0-OS4">
```
Comment (vmfo22) [inline plugin.xml:23]: @GFOutsystems the same as above

File: plugin.xml [hunk 3]
```diff
@@ -31,13 +31,13 @@
...
     <engines>
       <engine name="cordova" version=">=3.1.0" /><!-- Needs cordova/urlutil -->
+      <engine name="cordova-ios" version=">=4.0.0" />
     </engines>
 
     <!-- android -->
     <platform name="android">
         <js-module src="www/inappbrowser.js" name="inappbrowser">
             <clobbers target="cordova.InAppBrowser.open" />
-            <clobbers target="window.open" />
```
Comment (vmfo22) [inline plugin.xml:40]: @GFOutsystems isn't this a breaking change for us? What is the clobber that we're using in the OutSystems wrapper?

**Comment by GFOutsystems** [comment]
@CarlsCorrea @vmfo22 just resolved the comments. ;)

**Pull request merged**